bathbot/src/commands/osu/relax/top.rs

```diff
@@ -92,7 +92,9 @@ pub async fn top(orig: CommandOrigin<'_>, args: RelaxTop<'_>) -> Result<()> {
     };
 
     let Some(player) = player else {
-        return orig.error("Relax player not found").await;
+        return orig
+            .error(format!("Relax player `{}` not found", user.username))
+            .await;
     };
```

Comment:
Now that you changed it I noticed, would be nice if you handle this case in the `match` above immediately to make it more succinct but also avoid fetching the maps when the player wasn't available in the first place. So a match arm on `(scores, Some(player))` and `(score, None)`.